homeassistant/components/blebox/sensor.py

```diff
@@ -1,15 +1,23 @@
 """BleBox sensor entities."""
 from homeassistant.components.sensor import SensorDeviceClass, SensorEntity
 from homeassistant.config_entries import ConfigEntry
-from homeassistant.const import TEMP_CELSIUS
+from homeassistant.const import CONCENTRATION_MICROGRAMS_PER_CUBIC_METER, TEMP_CELSIUS
 from homeassistant.core import HomeAssistant
 from homeassistant.helpers.entity_platform import AddEntitiesCallback
 
 from . import BleBoxEntity, create_blebox_entities
 
-BLEBOX_TO_UNIT_MAP = {"celsius": TEMP_CELSIUS}
+BLEBOX_TO_UNIT_MAP = {
+    "celsius": TEMP_CELSIUS,
+    "concentration_of_mp": CONCENTRATION_MICROGRAMS_PER_CUBIC_METER,
```

Comment:
Instead of extending this we should use entity descriptions. Preferably we refactor this first in a separate PR.

Example:
https://github.com/home-assistant/core/blob/cf5a11a1e7c7ef49a439753f3bb708d10b2c22b9/homeassistant/components/airly/sensor.py#L64-L70